molecule/lint/yamllint.py

```diff
@@ -31,14 +31,32 @@
 
 class Yamllint(base.Base):
     """
-    `Yamllint`_ is the default projet linter.
+    `yamllint`_ is the default projet linter.
 
-    `Yamllint`_ is a linter for YAML files.  In addition to checking for syntax
-    validity, also checks for key repetition, and cosmetic problems such as
-    lines length, trailing spaces, and indentation.
+    `yamllint`_ is a linter for YAML files. In addition to checking for syntax
+    validity it also checks for key repetition as well as cosmetic problems such as
+    line length, trailing spaces, and indentation.
 
-    Additional options can be passed to `yamllint` through the options
-    dict.  Any option set in this section will override the defaults.
+    The default ``yamllint`` settings that ship with ``molecule`` are:
+
+    .. code-block: yaml
```

Comment:
I wonder if this won't be getting out-of-date very soon (extra maintenance issue for developers, misleading for users). Should it better to say here molecule spits out the defaults and also perhaps how? As far as I know, that means init'ing a scenario but maybe there is a separate command for that too?

Reply:
Very good point. I changed this to a link instead.